server/out/IonTextLexer.js

```diff
@@ -0,0 +1,643 @@
+// Generated from IonText.g4 by ANTLR 4.7.1
```

Comment:
These are ANTLR generated files. We need to figure out if we need the filed in the repository (not the best solution) or if we can run ANTLR on the grammar before we build the plugin and generate these files every time (better solution).

Reply:
I did not find a way to generate these files with ANTLR. I think I will simply create another folder for them.

Reply:
It will need java and the antlr tools installed. See https://medium.com/dailyjs/compiler-in-javascript-using-antlr-9ec53fd2780f

Lets move the files to a better folder for now and create an issue to see if we can have a better solution

Reply:
I created the issue